yarn.lock

```diff
@@ -10750,6 +10764,8 @@ __metadata:
   dependencies:
     "@babel/eslint-parser": 7.18.9
     "@firmnav/eslint-github-actions-formatter": 1.0.1
+    "@jeremybarbet/apple-api-types": 1.3.5
+    "@jeremybarbet/google-api-types": 1.3.5
```

Comment:
Cool, where are these used?

Reply:
It removes most of the types for Apple and all of the ones for Google. It's used [here](https://github.com/dooboolab/react-native-iap/pull/1797/files/beca8d361c6aedd4a19efc21dbecd4ceffe2a198#diff-38940e5890f175d4fbdec6f5a090d634063b6f7c65b2690a2f1ee30a91a2e830R2-R6). (`src/methods.ts`, make sure to click "Load Diff", Github is hiding the most important part..)